include/ignition/sensors/Sensor.hh

```diff
@@ -138,6 +138,10 @@ namespace ignition
       /// \return Name of sensor.
       public: std::string Name() const;
 
+      /// \brief FrameID.
+      /// \return FrameID of sensor.
+      public: std::string FrameID() const;
```

Comment:
how about `FrameId()` to be consistent with the [Id()](https://github.com/ignitionrobotics/ign-sensors/blob/ign-sensors3/include/ignition/sensors/Sensor.hh#L164) function name?

Add a simple test in [Sensor_TEST](https://github.com/ignitionrobotics/ign-sensors/blob/ign-sensors3/src/Sensor_TEST.cc#L64)?

Reply:
https://github.com/ignitionrobotics/ign-sensors/pull/195/commits/b2ad054c2e95b67eac01f7e7341f8df871425872